File: java/nopackage/basic/Sum.java

import java.util.Scanner;

class Sum {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int a = s.nextInt();
        System.out.print("Enter another number: ");
        int b = s.nextInt();
        int c = a + b;
        System.out.println("The sum is " + c);
    }
}

File: vb-net/console/basic/Sum.vb
Option Explicit On
Module Sum
    Sub Main()
        Dim a, b, c As Integer
        Console.Write("Enter a number: ")
        a = Console.ReadLine()
        Console.Write("Enter another number: ")
        b = Console.ReadLine()
        c = a + b
        Console.Write("The sum is " & c)
    End Sub
End Module